src/Data/Dictionary/DictionaryItem.php

```diff
@@ -0,0 +1,117 @@
+<?php
+declare(strict_types=1);
+namespace LORIS\Data\Dictionary;
+
+use \LORIS\Data\Scope;
+use \LORIS\Data\Type;
+use \LORIS\Data\Cardinality;
+
+/**
+ * A DictionaryItem represents a description of a type of data
+ * managed by LORIS.
+ *
+ * @license http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ */
+class DictionaryItem implements \LORIS\StudyEntities\AccessibleResource
+{
+    protected $name;
+    protected $description;
+    protected $scope;
+    protected $typ;
+
+    /**
+     * Construct a DictionaryItem with the given parameters
+     *
+     * @param string      $name  The field name of the dictionary item
+     * @param string      $desc  The dictionary item's description
+     * @param Scope       $scope The scope to which this DictionaryItem
+     *                           applies
+     * @param Type        $t     The data type of this dictionary item
+     * @param Cardinality $c     The data cardinality
+     */
+    public function __construct(
+        string $name,
+        string $desc,
+        Scope $scope,
+        Type $t,
+        Cardinality $c
```

Comment:
why single letter variables, i'm not a big fan and in the long run they always tend to be a hindrance (same goes for shortcuts like `cand` for `candidate`...)

Reply:
It's just the argument name in the constructor, the context only lasts long enough to assign it to a property right below.. the property name is the full word.